partcad/src/partcad/context.py

```
@@ -284,25 +284,25 @@ def _get_project_recursive(self, project, import_list: list[str]):
                     return result
         else:
             # Otherwise, iterate all subfolders and check if any of them are packages
-            if "import" in project.config_obj and not project.config_obj["import"] is None:
-                imports = project.config_obj["import"]
+            if "dependencies" in project.config_obj and not project.config_obj["dependencies"] is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)